source/core/mesh_controller.cc

```diff
@@ -0,0 +1,47 @@
+#include <core/mesh_controller.h>
+
+
+using namespace std;
```

Comment:
```suggestion
```
Let's avoid these namespace. We don't have that in the library (or we should not)